apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -147,6 +147,31 @@ const DialogAPIs = (): ReactElement => {
       },
     });
 
+  const SendMessageToParent_v2 = (): ReactElement =>
```

Comment:
I don't know much about this file: Why make _v2 versions of the functions rather than convert the existing functions to use the new dialog.* functions you're adding?

Reply:
(I'm not saying what you've done is wrong, I'd just love to learn more about what this file is used for)

Reply:
I think it would make sense to have two separate files for tasks and dialogs for testing. I am doing v2, so the e2e doesn't break. 
Once I am done with core work of changing APIs, I will create a PR for e2e testing. The benefit is, I would not have to deal with breaking e2e tests in every PR. 
If I don't do it, I would have to open a PR on host side, every time